src/View/InstallQuickInput.ts

```diff
@@ -168,11 +166,12 @@ class InstallQuickInput {
     });
     this.toolchainEnv = this.getToolchainEnvFromGlobal(state.selectedItem.label);
 
-    this.changeCurrentStepAfter(this.pickBackend.name, state as State);
+    this.changeCurrentStepAfter(this.pickBackend.name, state as InstallQuickInputState);
   }
 
-  async pickType(input: MultiStepInput, state: Partial<State>) {
-    this.changeCurrentStepBefore(this.pickType.name, state as State);
+  /* istanbul ignore next */
```

Comment:
I think some of codes can be tested, can I ask you why this function is excluded? :)

Reply:
Because this function is operated by user input.
`input.showQuickPick` function in L.191 can not be tested. :(

Reply:
I misunderstood that `MultiStepInput` is kind of vector or array something like that :(
Now I understand why these functions should be excluded :)
Thanks!